src/core_modules/capture-core/components/WidgetFeedback/WidgetFeedbackContent/WidgetFeedbackContent.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+import React, { type ComponentType } from 'react';
+import { spacersNum, colors } from '@dhis2/ui';
+import i18n from '@dhis2/d2-i18n';
+import { withStyles } from '@material-ui/core';
+import type { Props } from '../WidgetFeedback.types';
+
+const styles = {
+    container: {
+        padding: `0px ${spacersNum.dp16}px`,
+    },
+    unorderedList: {
+        paddingLeft: '15px',
+        marginTop: '0px',
+        lineHeight: '1.375',
+        fontSize: '14px',
+        color: colors.grey900,
+    },
+    noFeedbackText: {
+        color: colors.grey600,
+        paddingLeft: '15px',
+        fontWeight: 400,
+        fontSize: '14px',
+        marginTop: '0px',
+    },
+    listItem: {
+        marginBottom: '5px',
+        '&::marker': {
+            color: colors.grey500,
+        },
+    },
+};
+
+
+const WidgetFeedbackContentComponent = ({ displayText, displayKeyValue, classes }: Props) => {
+    if ((!displayText || displayText.length <= 0) && (!displayKeyValue || displayKeyValue.length <= 0)) {
```

Comment:
Hey, 
I think you can use [Optional chaining ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and the code will be more clear. WDYT?
`displayText?.length <= 0 && displayKeyValue?.length <= 0`
Thanks!

Reply:
Yep, definitely increased readability 👍